packages/contractkit/src/identity/odis/phone-number-identifier.ts

```diff
@@ -85,3 +90,10 @@ export function getPepperFromThresholdSignature(sigBuf: Buffer) {
     .digest('base64')
     .slice(0, PEPPER_CHAR_LENGTH)
 }
+
+/**
+ * Check if balance is sufficient for quota retrieval
+ */
+export function isSufficientBalanceForQuotaRetrieval(balance: BigNumber.Value) {
```

Comment:
I should have pointed this out in Amy's PR but the name here is misleading. Recommend `isBalanceSufficientForSigRetrieval`